app/services/ai_backend/anthropic.rb

```diff
@@ -28,6 +28,19 @@ def get_next_chat_message(&chunk_received_handler)
 
     response_handler = proc do |intermediate_response, bytesize|
       chunk = intermediate_response.dig("delta", "text")
+
+      input_tokens = intermediate_response.dig("message", "usage", "input_tokens")
+      if input_tokens
```

Comment:
stylistic feedback: since you are only using input_tokens here, you can combine these two lines:

```
if inputs_tokens = …
  @message …
end
```